### PR TITLE
Create and Edit entity client not working properly

### PR DESCRIPTION
**Pull request by Deepika1095**
Create and Edit entity client not working properly fix it

### CODE REVIEW

File: app/scripts/controllers/client/CreateClientController.js [hunk 1]
```diff
@@ -394,6 +395,7 @@
                 }
 
                 if(scope.first.inBusinessSince) {
+                    console.log("came");
```
Comment (hero78) [inline app/scripts/controllers/client/CreateClientController.js]: please remove logging

File: app/scripts/controllers/client/CreateClientController.js [hunk 2]
```diff
@@ -326,6 +326,7 @@
             };
 
             scope.submit = function () {
+                console.log(scope.formData);
```
Comment (hero78) [inline app/scripts/controllers/client/CreateClientController.js]: please remove logging

**Review by MageziArthur — COMMENTED** [review]
If you remove the logging all LGTM

File: app/scripts/controllers/client/CreateClientController.js (hunk 2 above)
Comment (Deepika1095) [reply app/scripts/controllers/client/CreateClientController.js]: Okay

**Comment by Deepika1095** [comment]
Removed it please approve again

**Review by hero78 — APPROVED** [review]

**Review by MageziArthur — APPROVED** [review]

**Pull request merged**